azure-pipelines.yml

```diff
@@ -4,6 +4,28 @@ trigger:
     include:
       - PHP-7.4
       - master
+  paths:
+    exclude:
+      - docs/*
+      - appveyor/*
+      - azure/*
+      - travis/*
+      - .appveyor.yml
+      - .editorconfig
+      - .gdbinit
+      - .gitattributes
+      - .gitignore
+      - .travis.yml
+      - CODING_STANDARDS.md
+      - CONTRIBUTING.md
+      - EXTENSIONS
+      - LICENSE
+      - NEWS
+      - README.REDIST.BINS
+      - README.md
+      - UPGRADING
+      - UPGRADING.INTERNALS
+      - azure-pipelines.yml
```

Comment:
We shouldn't be excluding the azure configuration.

I think generally this can be reduced to docs/, NEWS, UPGRADING, UPGRADING.INTERNALS, the rest doesn't change much.